blueprints/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_5_structural_analysis/formula_5_3a.py

```diff
@@ -0,0 +1,51 @@
+"""Formula 5.3a from NEN-EN 1992-1-1+C2:2011: Chapter 5 - Structural Analysis."""
+# pylint: disable=arguments-differ
+from blueprints.codes.eurocode.nen_en_1992_1_1_c2_2011 import NEN_EN_1992_1_1_C2_2011
+from blueprints.codes.formula import Formula
+from blueprints.type_alias import DIMENSIONLESS, KN
+from blueprints.validations import raise_if_negative
+
+
+class Form5Dot3ATransverseForceUnbracedMembers(Formula):
+    """Class representing formula 5.3a for the calculation of the transverse force for unbraced members, Hi.
+    See Figure 5.1 a1"""
+
+    label = "5.3a"
+    source_document = NEN_EN_1992_1_1_C2_2011
+
+    def __init__(
+        self,
+        theta_i: DIMENSIONLESS,
+        n: KN,
```

Comment:
I don't like the single letter symbols, why not 'normal_force' or 'axial_force'?

Reply:
The reason we're using the same notation in our formulas as in the Eurocode is because the entire group agreed on this during the last meeting.